python/nav/bootstrap.py

```diff
@@ -37,11 +37,18 @@ def bootstrap_django(caller=None):
 
     mydir = dirname(dirname(realpath(__file__)))
     sys.path.append(mydir)
-    logmsg = 'Bootstrapped at {}'.format(now())
-    if caller:
-        logmsg = logmsg + ', called by: ' + caller
-    print(logmsg)
-
-    if not RUN and not apps.ready:
-        django.setup()
-        RUN = True
+
+    print('Attempt to bootstrap by {} at {}'.format(caller, now()))
```

Comment:
No printing in production code, please :)

Reply:
Fixed in https://github.com/UNINETT/nav/pull/1684/commits/eb9e910710ed4363b50a1467ded32fd6d8e96926 .